1-exercises/A-this/exercise-3.js

```diff
@@ -21,4 +25,4 @@ console.log(
 bankAccount.withdraw(25);
 console.log(
   `Expected Result: 125. Actual result: ${bankAccount.currentAmount}`
-);
+);
```

Comment:
Your code is very readable since you are using prettier. But you need to put some comments your code to another people can easily understand it.